design/nfs/nfs-provisioner-controlled-by-operator.md

```diff
@@ -0,0 +1,115 @@
+# NFS Provisioner Controlled by Operator
+
+## Summary
+
+NFS Provisioner is a built in dynamic provisioner for Rook NFS. The functionality works fine but has a issue which the provisioner uses the underlying directory for each provisioned PV when provisioning two or more PV in the same share/export. This issue makes across provisioned PVs in the same share/export has the same data.
+
+This hierarchy is the current behaviour of NFS Provisioner when provisioning two PV in the same share/export.
+
+```console
+export
+├── sample-export
+|   ├── data (from PV-A)
+|   ├── data (from PV-B)
+|   ├── data (from PV-A)
+|   └── data (from PV-A)
+└── another-export
+```
+
+Both PV-A and PV-B uses the `sample-export` directory as their data location.
+
+This proposal is to make Rook NFS Provisioner to create sub-directory for each provisioned PV when provisioning two or more PV in the same share/export. So it will has a hierarchy like.
```

Comment:
it says we'll create a sub-directory when provisioning 2 or more PV in the same share/export.  Is that implying that we will not create the sub-directory when there's only 1 PV? If so, when the 2nd one is created, do we go back and create a sub-directory for the first PV?

I'm guessing we actually create the sub-directory for every single PV, even when there is only 1. Maybe we should clarify that here if it's truel.

Reply:
We create sub-directory for every PV. I think i need to edit this sentence. thanks!